inmaputil/wrapper.html

```diff
@@ -0,0 +1,27 @@
+{{ define "base" }}
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>InMap Web Interface</title>
+	<style>
+		html, body {padding: 0; margin: 2% 0; font-family: sans-serif;}
+		.container { max-width: 700px; margin: 0 auto; padding: 10px; }
+		blockquote { border-left: 5px solid #aaa; margin: .3em; color: #333; padding: 3px; }
```

Comment:
Can we make the text smaller for the block quote and use a fixed-width font for the editable fields, and maybe bold text for the flag names, with the goal being to make it more clear which information is most important and where data can be entered?